test/spmd/tensor/test_dtensor_ops.py

```diff
@@ -669,6 +670,11 @@ def wrapped(fn):
 
 
 def run_dtensor_crossref(test_case, func, args, kwargs):
+    run_bw = False
+    if hasattr(args[0], "requires_grad"):
```

Comment:
I think every tensor has `requires_grad` attribute? it's just that some is false and some is true. we should probably do it like this:

`if isinstance(args[0], torch.Tensor) and args[0].requires_grad`

Reply:
By default the value is False and instead of checking for if arg[0] is a Tensor, I check the presence of the attribute. I think we always want to run BW by default.

Reply:
got it, yeah make sense.